src/Spain/Mews.Fiscalizations.Spain/Model/Request/ForeignCustomer.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Mews.Fiscalizations.Core.Model;
+
+namespace Mews.Fiscalizations.Spain.Model.Request
+{
+    public class ForeignCustomer
+    {
+        public ForeignCustomer(Name name, ResidenceCountryIdentificatorType identificatiorType, String1To20 idNumber, Country country)
```

Comment:
Remove `ResidenceCountryIdentificatorType.NifVat` so you don't have to validate.
Country should not be spain